### DISCUSSION TITLE
Water vapor stratospheric bias: Q budget

### DISCUSSION THREAD

**Discussion by adamrher** [NCAR/amwg_dev#30] [TF1: Unified physics for L58/L93]
Is it the mountains? See change in DQCORE from L32 to L48 at 150hPa level [edit - units should say g/kg/day]:

![temp_map DQCORE 150hPa](https://user-images.githubusercontent.com/29961476/146650104-b5aaa38b-1e8c-4242-977e-1138ce21a184.png)

There is increased DQCORE over the convectively active west pacific in L48, which I would expect from the larger resolved vertical velocities in L48. But this seems to be more than offset by a drying in L48 over mountainous regions; the Andes and Himalays. This suggests to me that the terrain following coordinate in L32 is causing spurious mixing above the mountains, high up in the troposphere ... somehow ... I'm not sure what the coordinate surfaces look like up there, but if they are more curved in L32 than in L48, then the hyper-viscosity would be mixing along these curved surfaces (which is a well known issue). [edit: b/c this is cslam there is no hyper-viscosity on tracers ... so the issue may be implicit "horizontal" diffusion along coordinate surfaces]

@PeterHjortLauritzen if I wanted to plot the Eulerian reference grid used in the vertical remapping, I believe denoted "eta", how could I compute those? I'd like to overlay them in a longitude-height transect over the Andes.

@JulioTBacmeister if this is the fundamental reason that L32 has a nice looking tape recorder, then should we think twice about trying to reproduce the L32 solution in L58 via additional explicit vertical diffusion? Is the L32 spurious mixing fortuitous? Or is it a nuisance?  

@swrneale @islasimpson @cecilehannay

**Comment by adamrher** [comment]
Here are some additional diagnostics comparing L48 to L32 (parenthesis are the fractional change, averaged over +/-30˚ latitude).

DQCORE
[250 hPa (-8.22%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.DQCORE.L48sens.250hPa.pdf)
[150 hPa (-14.03%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.DQCORE.L48sens.150hPa.pdf)
[90 hPa (-44.82%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.DQCORE.L48sens.90hPa.pdf)
[Tropopause (-6.39%) note the tropopause is ~110 hPa over deep tropics, and declines to ~250 hPa in mid-to-high latitudes.](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.DQCORE.L48sens.tropopause.pdf)

PTEQ
[150 hPa (+14.04%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.PTEQ.L48sens.150hPa.pdf)
[90 hPa (+46.36%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.PTEQ.L48sens.90hPa.pdf)
[Tropopause (+7.07%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.PTEQ.L48sens.TROPOPAUSE.pdf)

Q
[250 hPa (-12.03%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.L48sens.250hPa.pdf)
[150 hPa (-21.77%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.L48sens.150hPa.pdf)
[90 hPa (-18.95%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.L48sens.90hPa.pdf) 

ZONAL-HEIGHT PLOTS (tendencies expressed as dln(Q)/dt ~ bar(dQ/dt)/bar(Q))
[Q](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.Q.frac.L48sens.pdf)
[DQCORE](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.DQCORE.L48sens.pdf), [LND](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.DQCORE.L48sens.LND.pdf), [OCN](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.DQCORE.L48sens.OCN.pdf)
[OMEGA](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.OMEGA.L48sens.pdf), [LND](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.OMEGA.L48sens.LND.pdf), [OCN](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.OMEGA.L48sens.OCN.pdf)
[PTEQ](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.PTEQ.L48sens.pdf)

**Comment by adamrher** [comment]
Here are some diagnostics for the increased tau (3600s->7200s) experiments. Both test and cntl use L58+zm2 (parenthesis are the fractional change, averaged over +/-30˚ latitude).

DQCORE
[250 hPa (-4.21%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.DQCORE.tausens.250hPa.pdf)
[150 hPa (+3.95%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.DQCORE.tausens.150hPa.pdf)
[90 hPa (+4.49%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.DQCORE.tausens.90hPa.pdf)
[Tropopause (+11.24%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.DQCORE.tausens.tropopause.pdf)

(Increasing the zm time-scale will increase resolved updrafts, and so DQCORE is the relevant variable to look at).

Q
[250 hPa (-1.15%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.tausens.250hPa.pdf)
[150 hPa (+2.18%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.tausens.150hPa.pdf)
[90 hPa (+6.48%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.tausens.90hPa.pdf)

Interesting how at 150 hPa the changes are more localized, coinciding with convective action centers, whereas at 90 hPa, the changes are spread out over the entire tropics. Suggests to me that to understand moisture changes at 90 hPa, 150 hPa changes may be more informative.

ZONAL-HEIGHT (tendencies expressed as dln(Q)/dt ~ bar(dQ/dt)/bar(Q)).
[Q](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.Q.tausens.pdf)
[PTEQ](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.PTEQ.tausens.pdf)
[DQCORE](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.DQCORE.tausens.pdf), [LND](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.DQCORE.tausens.LND.pdf), [OCN](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.DQCORE.tausens.OCN.pdf)
[OMEGA](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.OMEGA.tausens.pdf), [LND](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.OMEGA.tausens.LND.pdf), [OCN](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.OMEGA.tausens.OCN.pdf)

**Comment by adamrher** [comment]
Here are some diagnostics for the increased clubb diffusion experiments. Both test and cntl use L58+zm2 (parenthesis are the fractional change, averaged over +/-30˚ latitude).

PTEQ
[150 hPa (+0.42%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.PTEQ.clubbsens.150hPa.pdf)
[90 hPa (-3.63%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.PTEQ.clubbsens.90hPa.pdf)
[Tropopause (+0.97%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.PTEQ.clubbsens.TROPOPAUSE.pdf)

(explicit clubb diffusion is reflected in the physics tendencies, so PTEQ is the relevant variable to look at). Hell let look anyways

Q
[150 hPa (+6.30%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.clubbsens.150hPa.pdf)
[90 hPa (+4.30%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.clubbsens.90hPa.pdf)

I'm struck by the localized PTEQ changes at 150 hPa, while there are large changes in the Q field at 150 hPa, in particular the Southern Hemisphere where there are not PTEQ changes.

ZONAL-HEIGHT PLOTS (tendencies expressed as dln(Q)/dt ~ bar(dQ/dt)/bar(Q))
[Q](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.Q.frac.clubbsens.pdf)
[DQCORE](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.DQCORE.clubbsens.pdf)
[PTEQ](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.PTEQ.clubbsens.pdf)

**Reply by islasimpson** [reply]
I wonder if zonal mean cross sections of DQCORE and PTEQ as a function of latitude and height might be useful?  Particularly for ones where we see a big change e.g. maybe the difference where we see the biggest change.  We may be missing something by focusing on localized tendencies at an individual level and zonal mean cross sections might give a complementary view of what's going on.

**Reply by adamrher** [reply]
that's a good idea. I'll be working on lat-height and lon-height plots later today (I asked Pat for some help b/c he has those really nifty transect plots over the Andes).

**Reply by adamrher** [reply]
I took a shot at adding zonal-height plots. I think it all looks consistent, but I must admit I'm too tired to provide an analysis. To be continued, but feel free to opine.

**Reply by islasimpson** [reply]
Thanks.  So quite strong cancellation between the DTCORE and PTEQ.  Might be worth having one where they're summed up too, to see where there's a difference in the overall tendency so that we're not drawn toward big anomalies that are kind of irrelevant because they're being opposed by the other tendency?

**Reply by adamrher** [reply]
I took a look at total q-tendencies and they weren't that informative since they are so small at these climatological time-scales. I think climatological budgets are useful at looking at changes to individual terms. The drawback of only two terms (physics and dynamics) is you have to guess as to which one is being perturbed, since the other term will inevitably balance the perturbed term, over these long time-scales. With L32-L48, I'm quite confident it's the dynamics, and so I tend to focus on DQCORE there. With the clubb diffusion expts, we are clearly perturbing the physics and so that's a good guess as to the relevant term. And then with changing the convective time-scale, that's a bit of a curveball and I tend to believe the it's the dynamics responding to a less efficient convection scheme (this is a bit of a caricature understanding on my part; the tropics must achieve a radiative convective equilibrium, and if the param convection isn't aggressive enough, the sluggish "resolved convection" must take over to achieve balance). 

w.r.t the L32-L48 sensitivity, I've went ahead and added the meridional stream-function to the DQCORE and Q plots. I think they are interesting and offer some explanation for the DQCORE changes in those runs. There is more vigorous upwelling in the ITCZ at L48, but there is also a contraction, with anomalous downward motion over the two big negative anomalies in DQCORE (centered about +8N and -10S). I suspect these are associated with the reduction in DQCORE over topographic regions; the Central/South America/New Guinea/W. Indes/Asia. I am currently adding in a land mask / ocean mask to the zonal means, and will update if I find something interesting.

**Comment by PeterHjortLauritzen** [comment]
>@PeterHjortLauritzen if I wanted to plot the Eulerian reference grid used in the vertical remapping, I believe denoted "eta", how could I compute those? I'd like to overlay them in a longitude-height transect over the Andes.

Before entering physics the dynamical core remaps everything to the reference levels (A+B*PS). So you can get the Eulerian levels using surface pressure and the hybrid coefficients.

**Comment by adamrher** [comment]
Here are some diagnostics for L58+zm1 -> L58+zm2. If you recall, L58+zm1 has a [decent looking tape recorder](https://www.cgd.ucar.edu/staff/islas/watervapor/taperecorder.pdf). And so the goal is to quantify how much we need to change L58+zm2 to get something that looks like L58+zm1 (parenthesis below are the fractional change, averaged over +/-30˚ latitude, in going from zm1->zm2).

DQCORE
[150 hPa (-17.39%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.DQCORE.zm2sens.150hPa.pdf)
[90 hPa (-28.53%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.DQCORE.zm2sens.90hPa.pdf)
[Tropopause (-13.00%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.DQCORE.zm2sens.TROPOPAUSE.pdf)

Q
[150 hPa (-9.04%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.zm2sens.150hPa.pdf)
[90 hPa (-9.59%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.zm2sens.90hPa.pdf)

_**These results suggest we need to increase the q-tendencies by 15-30% to gives us a reasonable tape recorder in L58+zm2.**_ We may be able to get there by both increasing tau and adding background diffusion via clubb.

**Comment by adamrher** [comment]
Diagnostics for doubling the divergence damping (L58+zm2 -> L58+zm2+2xdivdamp).

Q
[90 hPa (+0.84%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.divsens.90hPa.pdf)
[150 hPa (-0.34%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.divsens.150hPa.pdf)
[250 hPa (+0.58%)](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_map.Q.divsens.250hPa.pdf)

ZONAL-HEIGHT PLOTS
[Q](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.Q.divsens.pdf)
[OMEGA](https://webext.cgd.ucar.edu/FSCAM/FWscHIST/L32_REF/12132021/temp_dzonal.OMEGA.divsens.pdf)

**Comment by adamrher** [comment]
This is a summary of last Monday's mtg (12/27). The main point I want to convey is that the cause of strat dry bias L32->L48 is due to the dynamics responding more aggressively to the detrainment from the ZM scheme. As I have shown many times, the ZM scheme acts the same in L32 and L48; it dumps the same amount of vapor into the UT, the buoyancy of cloud formation when that vapor condenses is therefore the same. But the resolved dynamics responds to this buoyancy with larger vertical velocities owing to the reduction in discretization errors in L48. This is illustrated by the DQCORE field (tendency due to moisture transport by the dycore), which shows an increase in moisture advection over the west pacific warm pool, and due to mass conservation, this creates an anomalous subsidence or drying tendency manifesting as an envelope around the west pacific (extending all way into the east pacific, i.e., the walker circulation). This anomolous subsidence dries out the lower stratosphere, as illustrated by the zonal height plot showing the fractional change in water vapor and change in meridional streamfunction.

![L48sens flyer](https://user-images.githubusercontent.com/29961476/148266828-d69367af-0d02-4398-b852-21ba284bece9.png)

Increasing the convective time-scale reverses this behavior by reducing detrainment into the UT, and therefore the buoyancy that the dynamics responds to.

**Comment by cecilehannay** [comment]
From Rolando:

Here is the most relevant article I found on validation of MLS h2o in the upper troposphere.

This validation is carried out over the Tibetan Plateau in NH summer, which is particularly relevant for wet season h2o biases.

MLS is apparently _dry_ compared to balloon-borne FPH in the upper troposphere (~100-300 hPa). See Figs. 7 and 8. RMS biases are large in the upper troposphere/lower stratosphere.

[Yan_etal-MLS-validation_AMT2016.pdf](https://github.com/NCAR/amwg_dev/files/7865108/Yan_etal-MLS-validation_AMT2016.pdf)
.
